WORKSPACE

```diff
@@ -0,0 +1,35 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```

Comment:
just want to check we can still build w/o bazel.. using Bazel at LinkedIn is still PITA

Reply:
Yeah, we can still build via `./scripts/bluepill.sh build` which still uses `xcodebuild`. I didn't want to _require_ `bazel`, just allow it. Homebrew uses `xcodebuild` too so I also didn't want to break that.